app/public/js/system/guiConfig.js

```diff
@@ -42,6 +42,102 @@ gui.App.on('reopen', function() {
     guiConfig.getGUI.show();
 });
 
+
+// Get the minimize event
+guiConfig.getGUI.on('minimize', function() {
+    if (window.localStorage.minimizeToTray === 'false' || !window.settings.traySupport()) return false;
```

Comment:
Can you do the long condition using the brackets `if ( blabla ) { // logic }` please